solutions/swb-reference/integration-tests/support/resources/datasets/dataset.ts

```diff
@@ -34,6 +34,9 @@ export default class Dataset extends Resource {
     return this._axiosInstance.delete(`/projects/${projectId}/datasets/${this._id}/relationships`);
   }
 
+  public async listAccessPermissions(): Promise<AxiosResponse> {
+    return this._axiosInstance.get(`/datasets/${this._id}/permissions`);
```

Comment:
This may or may not need to eventually be nested under `projects` for AuthZ, and as far as i can tell, the framework isn't set up to handle changes in paths very easily. Kevin and I have been talking about just getting code in for feature freeze and then evaluating / updating this thing so it better fits our needs. I can change it around for now though if you feel strongly about it.

Reply:
I think in the future, it would be better to maintain usage of `this._api` since there is a reason you put this test building method under this resource, but I won't make you do it now. 

In the case that you are explaining though, I think that's an argument for using `this._api` because we want route changes for resource/resource collections as a whole to require minimal refactor.